Dockerfile

```diff
@@ -26,7 +31,9 @@ ENV PORT= \
     RCON_ENABLED=true \
     RCON_PORT=25575 \
     QUERY_PORT=27015 \
-    TZ=UTC
```

Comment:
I believe this was a merge conflict issue, however I would like to keep this ENV :stuck_out_tongue:

Reply:
line 18 in the docker-compose.yml, this is the Dockerfile :slightly_smiling_face: